packages/hydrogen/src/build.ts

```diff
@@ -65,6 +66,7 @@ export const build: BuildV2 = async ({
     nodeVersion,
     env: spawnOpts.env || {},
     turboSupportsCorepackHome,
+    detectedLockfile: lockfilePath ? basename(lockfilePath) : undefined,
```

Comment:
It looks like that common logic should be exported from the build-utils package.